dojo_plugin/utils/dojo/__init__.py

```diff
@@ -193,7 +195,7 @@ def assert_one(query, error_message):
 
     dojo_kwargs = {
         field: dojo_data.get(field, getattr(import_dojo, field, None))
-        for field in ["id", "name", "description", "password", "type", "award"]
+        for field in ["id", "name", "description", "password", "type", "award", "icon"]
```

Comment:
This is meant to be covered by "award", which generically includes both belt awards and token awards. Displaying an icon should be determined based on that award.

Reply:
What is the intended type of "award"?  If it's just "blue belt" the mapping logic for str<->file_path becomes hard-coded somewhere or couples the award name to the filename.  No?